nautobot/core/views/generic.py

```diff
@@ -586,15 +586,16 @@ def post(self, request, **kwargs):
 
         if form.is_valid():
             logger.debug("Form validation was successful")
+            msg = ""
 
             try:
+                msg = f"Deleted {self.queryset.model._meta.verbose_name} {obj}"
```

Comment:
Does this need to be inside the try block for a reason? Or could this just be:

```suggestion
            msg = f"Deleted {self.queryset.model._meta.verbose_name} {obj}"

            try:
```